packages/api/src/context.ts

```diff
@@ -118,12 +118,14 @@ export function createTRPCContext(
           );
         }
 
+        let current_url = opts.req.url;
```

Comment:
```suggestion
        let currentUrl = opts.req.url;
```

we should use `camelCase` when using `TypeScript`

Reply:
Done [322d1a8](https://github.com/Blobscan/blobscan/pull/679/commits/322d1a871e09dde478b49c09f6b10138cf8bc5f5)